caddyhttp/httpserver/replacer.go

```diff
@@ -119,6 +122,19 @@ func NewReplacer(r *http.Request, rr *ResponseRecorder, emptyValue string) Repla
 
 				return requestReplacer.Replace(string(dump))
 			},
+			"{request_body}": func() string {
+				if !canLogRequest(r) {
+					return ""
+				}
+
+				body, err := readRequestBody(r)
+				if err != nil {
+					log.Printf("[WARNING] Cannot copy request body %v", err)
```

Comment:
Although technically an error, I don't think it merits cluttering up the process log in this case -- might as well just return an empty string?